nvflare/fuel/hci/conn.py

```diff
@@ -27,16 +28,20 @@
 
 LINE_END = "\x03"  # Indicates the end of the line (end of text)
 ALL_END = "\x04"  # Marks the end of a complete transmission (End of Transmission)
-
-
 MAX_MSG_SIZE = 1024
+MAX_DATA_SIZE = 512*1024*1024
+MAX_IDLE_TIME = 60
```

Comment:
I feel this 60 secs is a little too long. Any particular reason not to reject if idle for, say, 5 or 10 secs? We should set it to be as small as possible (so to minimize the time window for abusing) without screwing up the normal data exchange (with delay).

I also don't feel completely comfortable with hard-coded parameter values (MAX_DATA_SIZE, MAX_IDLE_TIME). Ideally we want to make them configurable. But currently we don't have a perfect way for that yet, especially for the low-level stuff.

Reply:
I was just trying to be conservative. Changed to 10 seconds.